package.json

```diff
@@ -56,6 +56,7 @@
         "./packages/labs/analyzer:build",
         "./packages/labs/cli:build",
         "./packages/labs/compiler:build",
+        "./packages/labs/compiler:build",
```

Comment:
this somehow got added again when there's one right above it, causing build breakage.

Reply:
hahaha! darn `git merge`! Thank you for catching that!

Reply:
![git-merge](https://github.com/lit/lit/assets/40413829/af414828-623a-4815-a05b-a7295960fc4e)